env/env.go

```diff
@@ -14,24 +14,24 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package util
+package env
 
 import (
-	"os"
+	. "sigs.k8s.io/release-utils/env/internal"
```

Comment:
Is the dot import really required here? Dot imports are discouraged and supposed to be used for tests in the case when there are circular dependencies ([Go wiki reference](https://github.com/golang/go/wiki/CodeReviewComments#import-dot)). I don't see that we have circular dependencies here, but I might be wrong.

Reply:
It's not required, changed as suggested :+1: